### DISCUSSION TITLE
An eBook on neutrino oscillations

### DISCUSSION THREAD

**Discussion by emptymalei** [thuliteio/doks#59] [Showcase]
Hi @h-enk , thanks for sharing this great project.
I made an eBook on the topic of neutrino oscillations using this theme. The experience was smooth.

Here is my eBook:
https://neutrino.leima.is/
![Screenshot_20201229-004329238](https://user-images.githubusercontent.com/663798/103249818-561ff880-4971-11eb-836e-bff7d9f86d1d.jpg)

**Comment by h-enk** [comment]
Hi @emptymalei , thanks and thank you for using Doks — nice to see what you've made out of it!

Are your OK with me adding the Neutrino Oscillations website to the Doks showcase?

**Reply by emptymalei** [reply]
Sure. 😸

**Reply by h-enk** [reply]
Now part of the Doks showcase: https://getdoks.org/showcase/neutrino-oscillations/

**Reply by emptymalei** [reply]
Thank you, @h-enk 

(And happy new year!)
